lib/throttle.js

```diff
@@ -25,6 +25,21 @@ module.exports = (api) => {
     };
   };
 
+  api.metasync.debounce = (
+    timeout,
+    fn,
+    args
+  ) => {
+    let timer = null;
+    return () => {
+      clearTimeout(timer);
+      timer = setTimeout(
+        () => (args !== undefined ? fn(...args) : fn()),
+        timeout
```

Comment:
@IvanProdaiko94 thanks, it indeed proves that the implementation from lodash works the same way.